quixstreams/app.py

```diff
@@ -199,9 +199,13 @@ def __init__(
             quix_config_builder = QuixKafkaConfigsBuilder(quix_sdk_token=quix_sdk_token)
 
         if broker_address and quix_config_builder:
-            raise ValueError("Cannot provide both broker address and Quix SDK Token")
+            raise ValueError(
+                "Cannot provide both `broker_address` and `quix_sdk_token`"
+            )
         elif not (broker_address or quix_config_builder):
-            raise ValueError("Either broker address or Quix SDK Token must be provided")
+            raise ValueError(
+                "Either `broker_address` or `quix_sdk_token` must be provided"
```

Comment:
Very minor change: could you please replace accent symbols with double-quotes`"` instead?

Reply:
Addressed the comment.